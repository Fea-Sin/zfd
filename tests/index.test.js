import pkg from '../package.json';

const testDist = process.env.LIB_DIR === 'dist';

describe('zfd dist files', () => {
  // https://github.com/ant-design/ant-design/issues/1638
  // https://github.com/ant-design/ant-design/issues/1968
  it('exports modules correctly', () => {
    const zfd = testDist ? require('../dist/zfd') : require('../components'); //eslint-disable-line
    expect(Object.keys(zfd)).toMatchSnapshot();
  });

  // https://github.com/ant-design/ant-design/issues/1970
  // https://github.com/ant-design/ant-design/issues/1804
  if (testDist) {
    it('should have zfd.version', () => {
      const zfd = require('../dist/zfd'); //eslint-disable-line
      expect(zfd.version).toBe(pkg.version);
    });
  }
});
