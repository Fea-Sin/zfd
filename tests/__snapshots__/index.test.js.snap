// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`zfd dist files exports modules correctly 1`] = `
Array [
  "Alert",
  "LocaleProvider",
]
`;
